mora/lora.py

```diff
@@ -238,45 +237,46 @@ class Connector:
     def __init__(self, **defaults):
         self.__validity = defaults.pop('validity', None) or 'present'
 
-        self.today = util.parsedatetime(
-            defaults.pop('effective_date', None) or util.today(),
+        self.now = util.parsedatetime(
+            defaults.pop('effective_date', None) or util.now(),
         )
-        self.tomorrow = self.today + datetime.timedelta(days=1)
+        self.now = self.now.replace(microsecond=0)
```

Comment:
Good question  — mainly because I was lazy and didn't want the long microseconds in the log and tests unless needed. In practice, it doesn't apply to the tests, though, and logging isn't _that_ important; I'll fix it.